### DISCUSSION TITLE
Wet Bulb Value

### DISCUSSION THREAD

**Discussion by jlrosssc** [dolezsa/thermal_comfort#297] [Q&A]
I downloaded and have been using thermal comfort and absolutely love the information it gives.  Has anyone used this data and calculated wet bulb from the derived data?  From an everyday use, wet bulb is the only value that this tool does not provide.

**Comment by jlrosssc** [comment]
I created this for Fahrenheit.  The humidity and temperature sensor references will have to be changed to fit your needs:

- platform: template
  sensors:
    wetbulb:
      friendly_name: "Wetbulb"
      entity_id:
      - sensor.outdoor_temperature
      - sensor.outdoor_humidity
      value_template: >-
        {% set T = states.sensor.outdoor_temperature.state | float %}
        {% set RH = states.sensor.outdoor_humidity.state | float %}
        {% set a = atan((0.151977*(RH + 8.31313659)**0.5)) | float %}
        {% set b = atan(T + RH) | float %}
        {% set c = atan(RH - 1.676331) | float %}
        {% set d = (RH**1.5)*0.00391838 | float %}
        {% set e = atan(RH * 0.023101)| float %}
        {% set Tw = (T * a + b - c + d * e) -4.686035 | float %}
        {{ Tw | round(2)}}
      unit_of_measurement: "°F"

**Reply by systems-monitor** [reply]
Doesn't wet bulb need wind and sun intensity?
Would love that in the main device
